cmd/nexusUpload_test.go

```diff
@@ -0,0 +1,44 @@
+package cmd
+
+import (
+	"github.com/SAP/jenkins-library/pkg/nexus"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestMavenEvaluateGroupID(t *testing.T) {
+	// This is just a temporary test to facilitate debugging
+	value, err := evaluateMavenProperty("../pom.xml", "project.groupId")
```

Comment:
shouldn't we have a test which mocks the maven execution?

Reply:
This test should be removed. Whether the evaluation of expressions actually works should be tested with an integration test for mavenExecute. I will see about the need to mock maven execution when I address your comments about test-coverage.